src/style.css

```diff
@@ -1 +1,88 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src:
+    url(../src/fonts/Avenir-Book.ttf) format('truetype'),
+    url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
```

Comment:
It's better to split your font-face declarations into two separate ones for different font weights. It's a common practice to have a separate @font-face declaration for each font weight and style combination. This way, the browser can select the correct font file based on the font-weight and font-style used in your CSS rules.